src/entities/Grant/types.ts

```diff
@@ -39,6 +40,13 @@ export enum NewGrantCategory {
   Sponsorship = 'Sponsorship',
 }
 
+export enum SubtypeAlternativeOptions {
+  All = 'all_grants',
+  Legacy = 'legacy',
+}
+
+export type SubtypeOptions = `${NewGrantCategory}` | `${SubtypeAlternativeOptions}`
```

Comment:
Maybe we can move this to proposal types and make this a `ProposalSubtype`? 🤔 Or `GrantSubtype`...